structuretoolkit/visualize.py

```diff
@@ -800,3 +800,73 @@ def _get_flattened_orientation(
     flattened_orientation[:3, :3] = _get_orientation(view_plane)
 
     return (distance_from_camera * flattened_orientation).ravel().tolist()
+
+
+def plot_isosurface(
+    mesh,
+    value,
+    structure_plot=None,
```

Comment:
Can you add type definitions here as well?

Reply:
Done